builds/securing-builds-by-strategy.adoc

```diff
@@ -0,0 +1,55 @@
+[id='securing-builds-by-strategy']
+= Securing builds by strategy
+include::modules/common-attributes.adoc[]
+:context: securing-builds-by-strategy
+toc::[]
+
+Builds in {product-title} are run in privileged containers. Depending on the
+build strategy used, this allows a user who can run builds to escalate their
+permissions on the cluster and host nodes. As a security measure, limit who can
+run builds and the strategy that is used for those builds. Custom builds are
+inherently less safe than Source builds, given that they can execute any code
+within a privileged container, and as such are disabled by default. Docker build
+permissions should also be granted with caution, as a vulnerability in the
+Dockerfile processing logic could result in a privileges being granted on the
+host node.
+
+By default, all users that can create builds are granted permission to use the
+Docker and Source-to-Image (S2I) build strategies. Users with *cluster-admin*
+privileges can enable the Custom build strategy, as referenced in the
+Restricting Build Strategies to a User Globally section.
```

Comment:
Does this section still exist?

Reply:
i believe so?

Reply:
I didn't see that string elsewhere on this PR or in the 4.0 branch, so it might have a different name.

Reply:
the names may not match, but it's the content that's further below in this section

Reply:
Yeah, it's there. Thanks!